specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json

```diff
@@ -5974,6 +6036,64 @@
         }
       }
     },
+    "CompositePathList": {
+      "description": "List of composite path",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/CompositePath"
+      }
+    },
+    "CompositePath": {
+      "type": "object",
+      "properties": {
+        "path": {
+          "type": "string",
+          "description": "The path for which the indexing behavior applies to. Index paths typically start with root and end with wildcard (/path/*)"
+        },
+        "order": {
+          "description": "Sort order for composite paths.",
+          "type": "string",
+          "enum": [
+            "Ascending",
+            "Descending"
+          ],
+          "x-ms-enum": {
+            "name": "CompositePathSortOrder",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SpatialSpec": {
+      "type": "object",
+      "properties": {
+        "path": {
+          "type": "string",
+          "description": "The path for which the indexing behavior applies to. Index paths typically start with root and end with wildcard (/path/*)"
```

Comment:
Add  "pattern":  property to define the regex. helps fail early.. probably at SDK layer.